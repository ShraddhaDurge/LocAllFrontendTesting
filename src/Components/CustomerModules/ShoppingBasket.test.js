import React from 'react';
import renderer from 'react-test-renderer';
import ShoppingBasket from './ShoppingBasket';

it('ShoppingBasket', () => {
    const tree=renderer.create(<ShoppingBasket />).toJSON();
    expect(tree).toMatchSnapshot();
})