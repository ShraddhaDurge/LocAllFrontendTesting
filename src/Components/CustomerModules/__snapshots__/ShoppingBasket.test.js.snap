// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShoppingBasket 1`] = `
<div
  className="MuiBox-root MuiBox-root-3"
>
  <div
    className="MuiBox-root MuiBox-root-4"
  >
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-6"
    >
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-6"
      >
        <div
          className="MuiPaper-root MuiCard-root makeStyles-card-1 MuiPaper-elevation1 MuiPaper-rounded"
        >
          <div
            className="MuiCardContent-root"
          >
            <div
              className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-5"
            >
              <div
                className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"
                style={
                  Object {
                    "margin": "0px 0px",
                    "padding": "0px 0px",
                  }
                }
              />
              <div
                className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4"
                style={
                  Object {
                    "align": "center",
                    "margin": "0px auto",
                    "padding": "20px 50px",
                  }
                }
              >
                <br />
                <div
                  className="MuiTypography-root MuiTypography-h3 MuiTypography-gutterBottom"
                />
                <div
                  className="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom MuiTypography-displayInline"
                />
                <div
                  className="MuiTypography-root MuiTypography-subtitle1 MuiTypography-gutterBottom MuiTypography-displayInline"
                >
                  per item
                </div>
                <div
                  className="MuiTypography-root MuiTypography-h6 MuiTypography-gutterBottom"
                >
                  <span
                    className="MuiBadge-root"
                  >
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                      data-testid="ShoppingBasketIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="m17.21 9-4.38-6.56c-.19-.28-.51-.42-.83-.42-.32 0-.64.14-.83.43L6.79 9H2c-.55 0-1 .45-1 1 0 .09.01.18.04.27l2.54 9.27c.23.84 1 1.46 1.92 1.46h13c.92 0 1.69-.62 1.93-1.46l2.54-9.27L23 10c0-.55-.45-1-1-1h-4.79zM9 9l3-4.4L15 9H9zm3 8c-1.1 0-2-.9-2-2s.9-2 2-2 2 .9 2 2-.9 2-2 2z"
                      />
                    </svg>
                     
                    <span
                      className="MuiBadge-badge MuiBadge-anchorOriginTopRightRectangle MuiBadge-colorSecondary"
                    >
                      1
                    </span>
                  </span>
                  <div
                    className="MuiButtonGroup-root"
                    role="group"
                  >
                    <button
                      className="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined"
                      disabled={false}
                      onBlur={[Function]}
                      onClick={[Function]}
                      onDragLeave={[Function]}
                      onFocus={[Function]}
                      onKeyDown={[Function]}
                      onKeyUp={[Function]}
                      onMouseDown={[Function]}
                      onMouseLeave={[Function]}
                      onMouseUp={[Function]}
                      onTouchEnd={[Function]}
                      onTouchMove={[Function]}
                      onTouchStart={[Function]}
                      tabIndex={0}
                      type="button"
                    >
                      <span
                        className="MuiButton-label"
                      >
                         
                        <svg
                          aria-hidden={true}
                          className="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M19 13H5v-2h14v2z"
                          />
                        </svg>
                      </span>
                    </button>
                    <button
                      className="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined"
                      disabled={false}
                      onBlur={[Function]}
                      onClick={[Function]}
                      onDragLeave={[Function]}
                      onFocus={[Function]}
                      onKeyDown={[Function]}
                      onKeyUp={[Function]}
                      onMouseDown={[Function]}
                      onMouseLeave={[Function]}
                      onMouseUp={[Function]}
                      onTouchEnd={[Function]}
                      onTouchMove={[Function]}
                      onTouchStart={[Function]}
                      tabIndex={0}
                      type="button"
                    >
                      <span
                        className="MuiButton-label"
                      >
                         
                        <svg
                          aria-hidden={true}
                          className="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                          />
                        </svg>
                      </span>
                    </button>
                  </div>
                </div>
                <button
                  className="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-button-2 MuiButton-containedPrimary"
                  disabled={false}
                  onBlur={[Function]}
                  onClick={[Function]}
                  onDragLeave={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  onKeyUp={[Function]}
                  onMouseDown={[Function]}
                  onMouseLeave={[Function]}
                  onMouseUp={[Function]}
                  onTouchEnd={[Function]}
                  onTouchMove={[Function]}
                  onTouchStart={[Function]}
                  tabIndex={0}
                  type="submit"
                >
                  <span
                    className="MuiButton-label"
                  >
                    Add to basket
                          
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                      data-testid="ShoppingBasketIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="m17.21 9-4.38-6.56c-.19-.28-.51-.42-.83-.42-.32 0-.64.14-.83.43L6.79 9H2c-.55 0-1 .45-1 1 0 .09.01.18.04.27l2.54 9.27c.23.84 1 1.46 1.92 1.46h13c.92 0 1.69-.62 1.93-1.46l2.54-9.27L23 10c0-.55-.45-1-1-1h-4.79zM9 9l3-4.4L15 9H9zm3 8c-1.1 0-2-.9-2-2s.9-2 2-2 2 .9 2 2-.9 2-2 2z"
                      />
                    </svg>
                  </span>
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div />
  </div>
</div>
`;
